### DISCUSSION TITLE
Search option

### DISCUSSION THREAD

**Discussion by gringojack** [mytourbook/mytourbook#1150] [Q&A]
Hi

Is it possible to list identhical tours (with same name and different date) and the "Average Speed" column using Search option, and how?
Thanks

**Comment by wolfgang-ch** [comment]
With the geo compare tool https://mytourbook.sourceforge.io/mytourbook/index.php/documentation/tools/tool-compare-tours-by-geo you can display the average speed, the next versiion will have some improvements https://github.com/mytourbook/mytourbook/issues/1139

![image](https://github.com/mytourbook/mytourbook/assets/1283445/ec4a596e-703d-4da1-a47e-fc22069a44ed)
